examples/TypeScriptMessaging/ios/Podfile

```diff
@@ -24,5 +24,6 @@ target 'TypeScriptMessaging' do
   use_flipper!()
   post_install do |installer|
     react_native_post_install(installer)
+    __apply_Xcode_12_5_M1_post_install_workaround(installer)
```

Comment:
Is this backwards compatible btw? I mean will it break for non M1

Reply:
Yes, it is backwards compatible and we must keep it. Or else the example apps won't run on M1.
This is a ruby script that runs over all the flipper pods and bumps their respective iOS target to 11.0.

This workaround should in theory be removed in react native once the underlying flipper issue is resolved. Pretty annoying tbh. The issue that @khushal87 faces are also due to some flipper workaround failing. 

There is also a patch in 0.68 to resolve a folly issue https://github.com/facebook/react-native/commit/60cef850bd3fd12c32ee1196bd19a559592d1465, which could be related.